units/generate_tasks.go

```diff
@@ -180,5 +199,5 @@ func parseProjects(jsonBytes []json.RawMessage) ([]model.GeneratedProject, error
 	if catcher.HasErrors() {
 		return nil, catcher.Resolve()
 	}
-	return projects, nil
+	return projects, catcher.Resolve()
```

Comment:
nit: you can remove the `if catcher.HasError()` block here too if you're going to return catcher.Resolve() anyway.

Reply:
Done.